lib/paperclip/deprecations.rb

```diff
@@ -0,0 +1,34 @@
+require 'active_support/deprecation'
+
+module Paperclip
+  class Deprecations
+    class << self
+      def check
+        warn_aws_sdk_v1 if aws_sdk_v1?
+        warn_outdated_rails if active_record_version < '4.2'
+      end
+
+      private
+
+      def active_record_version
+        ::ActiveRecord::VERSION::STRING
+      end
+
+      def aws_sdk_v1?
+        defined?(::Aws) && aws_sdk_version < '2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.